src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -367,11 +379,23 @@ trait SendsToAmplitudeWithEnvironment extends SendsToAmplitude {
 
     ("user_properties" ->
       ("channel" -> meta.normalizedChannel) ~
-        ("app_build_id" -> meta.appBuildId) ~
-        ("locale" -> meta.`environment.settings`.map(_.locale)) ~
-        ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
-        ("experiments" -> experimentsArray)) ~
-      ("user_id" -> getClientId)
+      ("sample_id" -> meta.sampleId) ~
+      ("app_build_id" -> meta.appBuildId) ~
+      ("app_name" -> meta.appName) ~
+      ("app_version" -> meta.appVersion) ~
+      ("locale" -> meta.`environment.settings`.map(_.locale)) ~
+      ("is_default_browser" -> meta.`environment.settings`.map(_.isDefaultBrowser)) ~
+      ("country" -> meta.geoCountry) ~
+      ("os" -> meta.os) ~
+      ("os_version" -> meta.`environment.system`.map(_.os.name)) ~
```

Comment:
app_version, country, os and os_version are all top-level amplitude fields that are already sent over (see lines 310-313)

Reply:
I see now the top-level fields are defined in https://amplitude.zendesk.com/hc/en-us/articles/204771828#keys-for-the-event-argument

Does seem like those should be sufficient. I'll remove these.